File: api/internal/models/planet.go
package models

import (
	"github.com/google/uuid"
)

type PlanetThreat string

const (
	RADIATION PlanetThreat = "RADIATION"
	TOXINS    PlanetThreat = "TOXINS"
	FREEZING  PlanetThreat = "FREEZING"
	HEAT      PlanetThreat = "HEAT"
)

type Planet struct {
	ID       uuid.UUID `gorm:"type:uuid;default:gen_random_uuid()"`
	SystemID uuid.UUID
	System   System
	Threat   string
}

File: bot/api/types/planet.py
from enum import StrEnum
from uuid import UUID

from pydantic import BaseModel


class PlanetThreat(StrEnum):
    RADIATION = "RADIATION"
    TOXINS = "TOXINS"
    FREEZING = "FREEZING"
    HEAT = "HEAT"


class Planet(BaseModel):
    id: UUID
    name: str
    system_id: UUID
    threat: PlanetThreat
